src/main/resources/application.properties

```
@@ -2,11 +2,17 @@
 spring.datasource.url=jdbc:mysql://178.21.11.17:3306/pet_shelter_db
 spring.datasource.username=zrazhevskiy
 spring.datasource.password=5T!65gr2
-telegram.bot.token=6417547561:AAHw2WpFDDiBfHINDN85j8oUpeIkRMQM1WA
+telegram.bot.token=6319251812:AAFZ3wxuS8ohsrgadxTtRnkXjMeZG9ne9m0
 
 spring.jpa.show-sql=true
-spring.jpa.hibernate.ddl-auto=update
+spring.jpa.hibernate.ddl-auto=none
```

```suggestion
spring.jpa.hibernate.ddl-auto=validate
```
для ликвибейз надо использовать validate